boards/arm/xenvm/doc/index.rst

```diff
@@ -0,0 +1,171 @@
+.. xenvm:
+
+ARMv8 Xen Virtual Machine
+#########################
+
+Overview
+********
+
+This board allows to run Zephyr as Xen guest on any ARMv8 board that supports
+ARM Virtualization Extensions.
+
+.. figure:: xen_project_logo.png
+   :align: center
+   :alt: XenVM
+
+   Xen virtual Guest (Credit: Xen Project)
+
+It provides minimal set of devices:
+
+* ARM Generic timer
+* GICv2
+* SBSA (subset of PL011) UART controller
+
+
+Hardware
+********
+Supported Features
+==================
+
+The following hardware features are supported:
+
++--------------+-------------+----------------------+
+| Interface    | Controller  | Driver/Component     |
++==============+=============+======================+
+| GIC          | virtualized | interrupt controller |
++--------------+-------------+----------------------+
+| SBSA UART    | emulated    | serial port          |
++--------------+-------------+----------------------+
+| ARM TIMER    | virtualized | system clock         |
++--------------+-------------+----------------------+
+
+The kernel currently does not support other hardware features on this platform.
+
+Devices
+========
+System Clock
+------------
+
+This board configuration uses a system clock frequency of 8.32 MHz. This is the
```

Comment:
8.32MHz, this is per SoC, not fixed.

Reply:
Right. Maybe this part of documentation is not clear? I tried to emphasis that this setting should be changed for user's actual hardware.